server/libs/userlibs.js

```diff
@@ -108,4 +119,33 @@ const updateUser = function (body) {
     })
 };
 
-module.exports = {register, login, updateUser}
\ No newline at end of file
+const logOut = function(request) {
+    return new Promise((resolve,reject) => {
+        let token = request.header('x-authentication');
+        let decoded = {};
+        try {
+            decoded = jwt.verify(token, secretKeys.tokenKey, process.env.JWT_SECRET);
```

Comment:
why we are verifying token in libs level

Reply:
We are not verifying token here. We are taking time at which this token was created from it's payload